src/main/java/uk/gov/justice/hmpps/prison/api/resource/BookingResource.java

```diff
@@ -361,15 +364,23 @@ public Alert updateAlert(@PathVariable("bookingId") @ApiParam(value = "bookingId
         @ApiResponse(code = 404, message = "Requested resource not found.", response = ErrorResponse.class, responseContainer = "List"),
         @ApiResponse(code = 500, message = "Unrecoverable error occurred whilst processing request.", response = ErrorResponse.class, responseContainer = "List")})
     @ApiOperation(value = "Offender alerts.", notes = "Offender alerts.", nickname = "getOffenderAlerts")
+    @ApiImplicitParams({
+        @ApiImplicitParam(name = "page", dataType = "java.lang.Integer", paramType = "query",
+            value = "Results page you want to retrieve (0..N). Default 0, e.g. the first page", example = "0"),
+        @ApiImplicitParam(name = "size", dataType = "java.lang.Integer", paramType = "query",
+            value = "Number of records per page. Default 10"),
+        @ApiImplicitParam(name = "sort", dataType = "java.lang.String", paramType = "query", allowableValues = "alertId,bookingId,alertType,alertCode,comment,dateCreated,dateExpires,active,ASC,DESC",
+            value = "Sort as combined comma separated property and uppercase direction. Multiple sort params allowed to sort by multiple properties. Default to dateExpires,DESC and dateCreated,DESC")})
```

Comment:
would have liked to set allowMultiple but then allowableValues go missing, also no idea how you can document the sort is as format "property,order" since the comma is also used to separate the allowed values.

So above is best comprise I could find